build-system/tasks/release/index.js

```diff
@@ -362,6 +371,27 @@ async function populateOrgCdn_(flavorType, rtvPrefixes, tempDir, outputDir) {
   logSeparator_();
 }
 
+/**
+ * Removes the V0 directory from all RTVs except for the Stable (01-prefixed) channel,
+ *
+ * @param {!Array<string>} rtvPrefixes list of 2-digit RTV prefixes to generate.
+ * @param {string} outputDir full directory path to emplace artifacts in.
+ * @return {Promise<void>}
+ */
+async function dedupV0_(rtvPrefixes, outputDir) {
+  await Promise.all(
+    rtvPrefixes
+      .filter((rtvPrefix) => V0_DEDUP_RTV_PREFIXES.has(rtvPrefix))
+      .map((rtvPrefix) => {
+        const rtvNumber = `${rtvPrefix}${VERSION}`;
+        const v0Path = path.join(outputDir, 'org-cdn/rtv', rtvNumber, 'v0');
+        return fs.rm(v0Path, {recursive: true});
```

Comment:
As an alternative to deleting individual sets of files in parallel, I wonder if it will be cleaner to use `del` and provide it with the full list of files to delete. E.g. this is what we do in `amp clean`:

https://github.com/ampproject/amphtml/blob/9610c72e3df790eead617329e59beb86dfea329c/build-system/tasks/clean.js#L71-L74

Reply:
Can't say that this looks particularly cleaner to me, they both look about the same and in that case I'd rather use one less dependency